x-pack/plugins/lens/public/editor_frame_service/editor_frame/config_panel/layer_panel.tsx

```diff
@@ -283,60 +289,72 @@ export function LayerPanel(
         className="lnsLayerPanel"
         style={{ visibility: isDimensionPanelOpen ? 'hidden' : 'visible' }}
       >
-        <EuiPanel data-test-subj={`lns-layerPanel-${layerIndex}`} paddingSize="s">
-          <EuiFlexGroup gutterSize="s" alignItems="flexStart" responsive={false}>
-            <EuiFlexItem grow={false} className="lnsLayerPanel__settingsFlexItem">
-              <LayerSettings
-                layerId={layerId}
-                layerConfigProps={{
-                  ...layerVisualizationConfigProps,
-                  setState: props.updateVisualization,
-                }}
-                activeVisualization={activeVisualization}
-              />
-            </EuiFlexItem>
-
+        <EuiPanel
+          data-test-subj={`lns-layerPanel-${layerIndex}`}
+          paddingSize="none"
+          hasBorder
+          hasShadow
+        >
+          <section className="lnsLayerPanel__layerHeader">
```

Comment:
As this is wrapping the header of the layer panel, I think a `header` element may be more semantically correct than the `section` element. If you wish to use both `header` and `section`, the `section` element should wrap the entire contents of the layer panel.